airflow/providers/odbc/hooks/odbc.py

```diff
@@ -34,7 +34,7 @@ class OdbcHook(DbApiHook):
 
     :param args: passed to DbApiHook
     :param database: database to use -- overrides connection ``schema``
-    :param driver: name of driver or path to driver. overrides driver supplied in connection ``extra``
+    :param driver: name of driver or path to driver. overrides default driver if set`
```

Comment:
```suggestion
    :param driver: name of driver or path to driver. overrides system default driver if set`
```
is that what is meant?

Reply:
Now it can be both - system and default_driver.